sql/2019/cdn/17_19.sql

```diff
@@ -2,50 +2,50 @@
 # 17_19: Percentage of HTTPS responses by protocol
 SELECT
   client, cdn, firstHtml,
-  countif(ifnull(a.protocol, b.protocol) = 'HTTP/0.9') as http09,
-  countif(ifnull(a.protocol, b.protocol) = 'HTTP/1.0') as http10,
-  countif(ifnull(a.protocol, b.protocol) = 'HTTP/1.1') as http11,
-  countif(ifnull(a.protocol, b.protocol) = 'HTTP/2') as http2,
-  countif(ifnull(a.protocol, b.protocol) not in ('HTTP/0.9', 'HTTP/1.0', 'HTTP/1.1', 'HTTP/2')) as http_other,
-  countif(isSecure or ifnull(a.protocol, b.protocol) = 'HTTP/2') as tls_total,
-  round(100*countif(ifnull(a.protocol, b.protocol) = 'HTTP/0.9')/count(0), 2) as http09_pct,
-  round(100*countif(ifnull(a.protocol, b.protocol) = 'HTTP/1.0')/count(0), 2) as http10_pct,
-  round(100*countif(ifnull(a.protocol, b.protocol) = 'HTTP/1.1')/count(0), 2) as http11_pct,
-  round(100*countif(ifnull(a.protocol, b.protocol) = 'HTTP/2')/count(0), 2) as http2_pct,
-  round(100*countif(ifnull(a.protocol, b.protocol) not in ('HTTP/0.9', 'HTTP/1.0', 'HTTP/1.1', 'HTTP/2')) /count(0), 2) as http_other_pct,
-  round(100*countif(isSecure or ifnull(a.protocol, b.protocol) = 'HTTP/2')/count(0), 2) as tls_pct,
-  count(0) as total
+  countIF(IFNULL(a.protocol, b.protocol) = 'HTTP/0.9') AS http09,
```

Comment:
Shou

Reply:
I've turned off function capitalisation checking (rule L030) as we don't use them for our UDF function names. Unfortunately it sees `COUNTIF` as a function rather than a keyword and can't differentiation capitalisation rules between inbuilt ones and UDFs.

Anyway fixed this oversight in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2298/commits/08c5198f15d32146fc7bb8d424bdc282f614f9d5 but unfortunately errors like this won't be picked up automatically.